### PR TITLE
Began to style components; fixed up various API calls

### PR DESCRIPTION
**Pull request by misspia**

### CODE REVIEW

File: src/profile/profile.styles.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import styled, { css } from 'react-emotion';
+
+export const TabContainer = styled('div')`
+  display: flex;
+  margin-bottom: 10px;
+`;
+
+export const TabHeader = styled('div')`
+  border-radius: 0.4em;
+  background-image: linear-gradient(-225deg, #22E1FF 0%, #1D8FE1 48%, #625EB1 100%);
```
Comment (misspia) [inline src/profile/profile.styles.jsx:11]: Do you think we'd benefit from having a `Themes` folder and a file called `colors.js` that we can import to our style sheets? I've seen that being used in some big boilerplates

Comment (misspia) [reply src/profile/profile.styles.jsx:11]: `Themes` may also store some images, icons, `metrics` (like heights, widths, font sizes etc that we use a lot)

File: src/profile/stats/stats.container.jsx [hunk 2]
```diff
@@ -3,12 +3,26 @@ import Stats from './stats.jsx';
...
 const stateToProps = (state) => ({
 	selectedId: state.list.selectedId,
-	hp: state.profile.stats.hp,
-	attack: state.profile.stats.attack,
-	defense: state.profile.stats.defense,
-	specialAttack: state.profile.stats.specialAttack,
-	specialDefense: state.profile.stats.specialDefense,
-	speed: state.profile.stats.speed,
+	stats: [
+		{ 'name': 'hp',
```
Comment (misspia) [inline src/profile/stats/stats.container.jsx:7]: I wonder if itd be better to hide this in the reducer

File: src/profile/stats/stats.styles.jsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+export const StatsContainer = styled('div')`
+  background-image: linear-gradient(-225deg, #a7a6cb 0%, #8989ba 52%, #8989ba 100%);
+  padding: 15px 20px;
+  color: white;
+  border-radius: 0.4em;
+`;
+
+export const Row = styled('div')`
+  display: flex;
+  justify-content: space-evenly;
```
Comment (misspia) [inline src/profile/stats/stats.styles.jsx:13]: woah didnt know this existed LOL

File: src/profile/stats/stats.styles.jsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+export const ProgressName = styled('div')`
+  flex: 1;
+  text-align: right;
+  text-transform: uppercase;
+  font-family: 'Dosis', sans-serif;
+  font-weight: 700;
+`;
+
+export const ProgressBar = styled('div')`
+  font-family: 'Concert One', cursive;
```
Comment (misspia) [inline src/profile/stats/stats.styles.jsx]: yeah def feel we should have some centralized file/folder for storing things like this

File: src/profile/profile.styles.jsx (hunk 1 above)
Comment (iFallUpHill) [reply src/profile/profile.styles.jsx:11]: Yeah, I think we should. Was just kind of playing around with colors / showing off styles.

File: src/profile/stats/stats.container.jsx (hunk 2 above)
Comment (iFallUpHill) [reply src/profile/stats/stats.container.jsx:7]: Probably, wasn't sure where the best place to hide it was.

File: src/profile/stats/stats.styles.jsx (hunk 3 above)
Comment (iFallUpHill) [reply src/profile/stats/stats.styles.jsx:13]: boba has made me a flexbox expert LOL

File: src/profile/stats/stats.styles.jsx (hunk 4 above)
Comment (iFallUpHill) [reply src/profile/stats/stats.styles.jsx]: 👍

**Comment by iFallUpHill** [comment]
LGTM

**Pull request merged**